lex/iri_test.go

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017-2022 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package lex
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type testCases struct {
```

Comment:
this should be `testCase` instead of `testCases` because it just one test case.

Reply:
Done @mangalaman93